README.md

```diff
@@ -1,36 +1,70 @@
+[![Open in Visual Studio Code](https://classroom.github.com/assets/open-in-vscode-2e0aaae1b6195c2367325f4f02e2d04e9abb55f0b24a779b69b11b9e10269abc.svg)](https://classroom.github.com/online_ide?assignment_repo_id=15555388&assignment_repo_type=AssignmentRepo)
 # SE_Day1
 Software Engineering Day1 Assignment
 
 #Part 1: Introduction to Software Engineering
 
 Explain what software engineering is and discuss its importance in the technology industry.
 
+software engineering is the systematic application of engineering principles, methods and tools to the development and maintenance of high-quality software principles.
+software engineering helps in the creating of software applications and systems that power various aspects of modern life such as transport and logistics, healthcare, banking and finance and etc.
```

Comment:
i. Establishment of software engineering as a discipline in the 1960s. 
ii. Advent of structured programming in the 1970s.
iii. Rise of agile methodologies in the 2000s.

Reply:
1.cloud computing - This is the on demand delivery  of storage , compute ,network  over the internet.
2.Artificial intelligence - This is the simulation of human intelligence  in machines
3.Development of microservice architecture- involves breaking down applications into smaller independent  services that can be deployed  and scaled independently.

Reply:
1.AI- simulates human intelligence
2.Machine Learning-creates learning platforms
3. Cloud Computing-offers storage

Reply:
1- High-Level Languages (1950s-1960s): Languages like FORTRAN and COBOL made programming easier and more efficient.

2 - Structured Programming (1960s-1970s): Introduced modular and organized code to improve readability and reduce errors.

3 - Agile Methodology (2001): Brought flexible, iterative development focused on collaboration and adapting to change.